front/app/containers/Admin/projects/project/files/components/FilesUpload/components/FileDropzone.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```

Comment:
Note: Eventually we might want to generalize this "multi-file upload" component so we can use it for other file uploads across the admin interface, but for now this is specifically for the new "Project Files" tab + blended data feature.